app/src/main/res/values-tr/strings.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
+
+  <string name="widget_label" translatable="false">Stocks Tracker Aracı</string>
+  <string name="news_endpoint" translatable="false">https://news.google.com/</string>
+  <string name="suggestions_endpoint" translatable="false">https://query2.finance.yahoo.com/v1/finance/</string>
+  <string name="yahoo_endpoint" translatable="false">https://query1.finance.yahoo.com/v7/finance/</string>
+  <string name="historical_data_endpoint" translatable="false">https://query1.finance.yahoo.com/v8/finance/</string>
+  <string name="github_endoint" translatable="false">https://api.github.com/</string>
+  <string name="checkout_open_source" translatable="false">https://github.com/premnirmal/StockTicker</string>
+  <string name="privacy_policy_url" translatable="false">https://github.com/premnirmal/StockTicker/blob/master/PRIVACY_POLICY.md</string>
+  <string name="dotdotdot" translatable="false">…</string>
```

Comment:
all of these non-translatable strings should not be present in this file, can they be removed? (lines 4-12)